docs/vNext/Ethereum_Configuration.md

```diff
@@ -25,7 +23,7 @@ The page covers the following aspects of using the Ethereum adapter:
 
 # Assembling the Network Configuration File
 
-The JSON network configuration file of the adapter essentially defines which contract are expected to be on the network and which account the adapter should use to deploy the pointed contracts and which account use to invoke them. Contract necessary for the benchmark are automatically deployed at benchmark startup while the special `registry` contract needs to be already deployed on the network. ABI and bytecode of the registry contract are reported in the src/contract/ethereum/registry/registry.json.
+The JSON network configuration file of the adapter essentially defines which contract are expected to be on the network and which account the adapter should use to deploy the pointed contracts and which account use to invoke them. 
```

Comment:
`which contract are` should be `which contracts are`

Reply:
done